web/client/utils/__tests__/DOMUtil-test.js

```diff
@@ -0,0 +1,27 @@
+
+/*
+ * Copyright 2019, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const expect = require('expect');
```

Comment:
also here you can use import syntax
import expect from 'expect';
import { scrollIntoViewId } from '../DOMUtil';